types/react-navigation/index.d.ts

```diff
@@ -18,6 +18,7 @@
 //                 Edward Sammut Alessi <https://github.com/Slessi>
 //                 Jérémy Magrin <https://github.com/magrinj>
 //                 Luca Campana <https://github.com/TizioFittizio>
+//                 Ullrich Schaefer <https://github.com/stigi>
```

Comment:
Confirming you know that adding your name here makes you a maintainer, and you'll receive further notifications for any PRs and issues for react-navigation

Reply:
@iRoachie That's cool with me. I'm using `react-navigation` in all of my RN projects, so I'm more than happy to get a bit involved. Just started migrating from Flow to TypeScript where contributing to the types seems much easier. Good to Go?

Reply:
Great!